app/controllers/commands_controller.rb

```diff
@@ -20,7 +20,16 @@ def index
         @commands = @commands.where(project_id: project_id)
       end
     end
-    @pagy, @commands = pagy(@commands, page: page, items: 15)
+    respond_to do |format|
+      format.html do
+        @pagy, @commands = pagy(@commands, page: page, items: 15)
+      end
+
+      format.json do
+        render json: {commands: @commands.as_json}
```

Comment:
- use pagination since this will blow up when someone has 10k commands :)
 - use `render_as_json :commands, @commands`